doc/developers/bug_triaging.rst

```diff
@@ -44,6 +43,13 @@ The following actions are typically useful:
    <http://gael-varoquaux.info/programming/technical-discussions-are-hard-a-few-tips.html>`_
    explores how to lead online discussions in the context of open source.
 
+Working on PR to help review
```

Comment:
I feel like this isn't a triage-specific item?
As the text below suggest, everyone is welcome to review. No strong opinion

Reply:
if we keep it: PR -> PRs

Reply:
Indeed this is not triage-specific. The previous section also isn't triage specific.
My idea was to gather all triage-specific actions in the section "Triaging operations for members of the core and triage teams", but also clarify that all contributors are allowed to perform some triaging actions without being in the github team.
Should I put more emphasis on that?

Reply:
ok, that's fine then